packages/devtools_app/lib/src/memory/memory_allocation_table_view.dart

```diff
@@ -229,13 +226,15 @@ class AllocationTableViewState extends State<AllocationTableView>
           controller.searchMatchMonitorAllocationsNotifier,
     );
 
+    final ScrollController scroller = ScrollController();
+
     return Split(
       initialFractions: const [0.8, 0.2],
       minSizes: const [200, 0],
       axis: Axis.vertical,
       children: [
         controller.allocationsFieldsTable,
-        trackerData.createTrackingTable(context, controller),
+        trackerData.createTrackingTable(context, controller, scroller),
```

Comment:
instead of having this be a method on TreeTracker, create a widget class TrackingTable and use that here

Reply:
I'll put TODOs for this and the other function to be a class and move to a new file outside of MemoryTrackerModel